dist/rpm/phoebe.spec

```
@@ -0,0 +1,74 @@
+#
+# spec file for package phoebe
+#
+# Copyright (c) 2021 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+Name:           phoebe
+Version:        0.0.4
+Release:        0.3

```

At least for building on OBS, the release tag is ignored and set by the build system.
```suggestion
Release:        0
```